sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts

```diff
@@ -76,6 +97,9 @@ export const validateMptPAT = (): void => {
     if (Date.now() >= claims.exp! * 1000) {
       exitWithFailureMessage(ServiceErrorMessageConstants.EXPIRED_MPT_PAT_ERROR);
     }
+    if (result!.accountId !== claims!.aid) {
+      handleFailure?.(Constants.WORKSPACE_MISMATCH_ERROR);
```

Comment:
This should pass the reason why it is terminating, the message that needs to be displayed is the responsibility of the callback function